src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -362,6 +366,16 @@ def snapshot(self) -> list[Table]:
         """
         return self._snapshot(partial(self._try_load), partial(self._crawl))
 
+    def is_view(self, schema_name: str, table_name: str) -> bool:
+        table = self._try_load_one(schema_name, table_name)
+        return False if table is None else table.what == What.VIEW
```

Comment:
`skip_table_or_view` would `ALTER TABLE` resulting in an error when table is not a view and is not found. just expose `try_load_one`, remove this method, and incorporate this logic in `skip_table_or_view`

Reply:
done

Reply:
done